### PR TITLE
Get last season from db

### PR DESCRIPTION
**Pull request by healfy**
Add test and better handling of last season

### CODE REVIEW

File: W3ChampionsStatisticService/Matches/MatchRepository.cs [hunk 1]
```diff
@@ -241,4 +242,9 @@ public Task DeleteOnGoingMatch(string matchId)
         return _cache.CountOnGoingMatches(gameMode, gateWay, map, minMmr, maxMmr);
     }
 
+    public Task<Season> LoadLastSeason()
```
Comment (gustav87) [inline W3ChampionsStatisticService/Matches/MatchRepository.cs:245]: Can you query the Season collection for the current season? Here's what it contains.
```
W3Champions-Statistic-Service> db.Season.find()
[
  { _id: 0 },  { _id: 1 },
  { _id: 2 },  { _id: 3 },
  { _id: 4 },  { _id: 5 },
  { _id: 6 },  { _id: 7 },
  { _id: 8 },  { _id: 9 },
  { _id: 10 }, { _id: 11 },
  { _id: 12 }, { _id: 13 },
  { _id: 14 }, { _id: 15 },
  { _id: 16 }, { _id: 17 },
  { _id: 18 }
]
```

File: W3ChampionsStatisticService/Matches/MatchesController.cs [hunk 2]
```diff
@@ -26,8 +26,13 @@ public MatchesController(IMatchRepository matchRepository, MatchQueryHandler mat
         int offset = 0,
         int pageSize = 100,
         GameMode gameMode = GameMode.Undefined,
-        int season = 18)
+        int season = 0)
```
Comment (gustav87) [inline W3ChampionsStatisticService/Matches/MatchesController.cs:29]: There is a season 0. Either set default null or perhaps -1.
```suggestion
        int? season = null)
```

File: W3ChampionsStatisticService/Matches/MatchRepository.cs (hunk 1 above)
Comment (healfy) [reply W3ChampionsStatisticService/Matches/MatchRepository.cs:245]: sorry, didn't understand, could you please add some details?
```c#
mongoCollection.AsQueryable().OrderByDescending(c => c.Id).FirstOrDefaultAsync();
```
returns only one object, as far I understand

File: W3ChampionsStatisticService/Matches/MatchesController.cs (hunk 2 above)
Comment (healfy) [reply W3ChampionsStatisticService/Matches/MatchesController.cs]: fixed

File: W3ChampionsStatisticService/Matches/MatchRepository.cs (hunk 1 above)
Comment (gustav87) [reply W3ChampionsStatisticService/Matches/MatchRepository.cs:245]: I think I'm just C# illiterate! This already queries the Season collection, right?

**Review by gustav87 — APPROVED** [review]

File: W3ChampionsStatisticService/Matches/MatchRepository.cs (hunk 1 above)
Comment (healfy) [reply W3ChampionsStatisticService/Matches/MatchRepository.cs:245]: yeah

**Pull request merged**